frontend/resourceadm/pages/ErrorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { CenterContainer } from 'resourceadm/components/CenterContainer';
+import classes from './ErrorPage.module.css';
+import { Footer } from 'resourceadm/components/Footer';
+
+export const ErrorPage = () => {
+  return (
+    <>
+      <div className={classes.pageWrapper}>
+        <CenterContainer>
+          <h2>
```

Comment:
Her skipper du ```<h1 />```. Dette vil gi en warning i Wave om at du har skippet et headerlevel. Som jeg nevnte sist gang, så må headerlevelsene komme i en rekkefølge som er naturlig fra 1 og nedover. Anbefaler å ta en titt på Wave og laste det ned slik at du kan sjekke for feil osv. Linken er her: 
https://wave.webaim.org/extension/